src/analysisd/decoders/security_configuration_assessment.c

```diff
@@ -1556,7 +1561,9 @@ static void FillScanInfo(Eventinfo *lf,cJSON *scan_id,cJSON *name,cJSON *descrip
             sprintf(value, "%d", failed->valueint);
         } else if (failed->valuedouble >= 0) {
             sprintf(value, "%lf", failed->valuedouble);
-        } 
+        } else {
+            return;
```

Comment:
This is an unexpected condition: invalid value. Please print a debugging log here.